eoscostarica/eos-rate
# fix(ci): fix eslint and react-scripts duplicated dependencies

### Steps to test
- Remove `node_modules`
- Install all dependencies.
- No warnings or errors should come up.
- Run the project to make sure everything's ok `yarn start`
